src/Components/ArticlePage/ArticlePage.jsx

```diff
@@ -0,0 +1,32 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { Article } from "./Article";
+import { getArticle } from "../../api";
+import { Comments } from "./Comments";
+import { Suggested } from "./Suggested";
+
+export const ArticlePage = () => {
+  const [article, setArticle] = useState({});
+  const { article_id } = useParams();
+  //loading and errors
```

Comment:
Not sure why this comment needs to be here as it doesnt explain anything that is unreadable and is referencing something that doesnt exist